app/controllers/organisers_controller.rb

```diff
@@ -2,12 +2,58 @@ class OrganisersController < ApplicationController
    before_action :require_organiser
 
   def index
-    @organisers = User.where(role: 1)
+    display_organisers
+  end
+  
+  def create
+    @user = User.find_by(email: params[:email])
+    if @user.save
+       @user = User.new(email: params[:email])
+        OrganisersMailer.organisers_add_email(@user).deliver_now
+       redirect_to organisers_path, notice: 'Instructions have been sent to the email'
+    else
+       flash.now[:alert] = "User exist"
+       display_organisers
+       render "index"
+
+    end
+  end
+
+  def edit
```

Comment:
can you move `edit` and `update` to another controller like organiser_registration or something? later we want to use `edit` and `update` in this controller for editing organiser which was already created